cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -33,6 +33,11 @@ else()
   set(configure_command "./config")
 endif()
 
+# Pass fPIC through
+if(";${CMAKE_CXX_FLAGS};" MATCHES ";-fPIC;")
```

Comment:
I do not think it is safe to send all CMAKE_CXX_FLAGS because ./config uses its own inbuilt flags that could conflict, such as -d or -v.
I chose -fPIC because it was causing me issues.